core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -139,6 +198,144 @@ var StoryEditorPage = function() {
     closeSaveModalButton.click();
     waitFor.pageToFullyLoad();
   };
+
+  this.expectSaveStoryDisabled = function() {
+    return expect(
+      saveStoryButton.getAttribute('disabled')).toEqual('true');
+  };
+
+  this.expectDisplayUnreachableChapter = function() {
+    return expect(disconnectedChapterWarning.isPresent()).toBe(true);
+  };
+
+  this.setChapterExplorationId = function(explorationId) {
+    waitFor.visibilityOf(
+      explorationIdInput,
+      'ExplorationIdInput takes too long to be visible'
+    );
+
+    explorationIdInput.sendKeys(explorationId);
+    waitFor.elementToBeClickable(
+      explorationIdSaveButton,
+      'ExplorationIdSaveButton takes too long to be visible'
```

Comment:
Instead of `visible`, maybe `clickable` to better align with what we're actually waiting for?

Reply:
Updated